browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -1969,12 +1974,12 @@ L.CanvasTileLayer = L.Layer.extend({
 
 	_onCalcFunctionUsageMsg: function (textMsg) {
 		var pos = this._lastVisibleCursorRef._northEast;
-		this._map.uiManager.showDocumentTooltip(textMsg, $('.leaflet-layer'), pos);
+		this._map.uiManager.showFormulaTooltip(textMsg, $('.leaflet-layer'), pos);
```

Comment:
btw, I think we can remove all these: `$('.leaflet-layer')` and just get that element directly inside called function.
It is always the same as these tooltips are for "canvas area", no need for a parameter now.